### DISCUSSION TITLE
Support for manually specifying the URL of the PR for the ci command

### DISCUSSION THREAD

**Discussion by cangSDARM** [danger/danger-js#1183] [General]
We have several repositories, one contains `dangerflie.js` and the other contains project files.
Gitlab allows multi-project to trigger others Pipeline, we use it to deal with dangerfile related things.
But the current danger does not seem to support this feature. Pipeline is successfully triggered, but the context obtained by dangerjs is not the context of the project we want.
So can you add related features for `danger ci` to support this scenario, such as `danger ci --for-remote-pr  https://gitlab.com/ower/repo/~/merge_requests/xx`?
Or give us some advices?

https://docs.gitlab.com/ee/ci/pipelines/multi_project_pipelines.html

**Comment by orta** [comment]
I don't use GitLab, so I'm afraid you'll probably have to think out a good way to make this work on your own I'm afraid

**Reply by cangSDARM** [reply]
Will this happen in the future? Support url for the ci command, like `danger ci --for-remote-pr REMOTE_URL`

**Comment by cangSDARM** [comment]
We use *settings/ci_cd/General pipelines* to configure the remote yml file to solve this problem.

But I am still curious about whether we plan to add this feature in the future.
